### PR TITLE
add --dry-run arg to rosdistro_reformat

### PR DESCRIPTION
**Pull request by mintar**

### CODE REVIEW

File: scripts/rosdistro_reformat [hunk 1]
```diff
@@ -37,7 +37,7 @@ import argparse
 import os
 import sys
 
-from rosdistro.verify import reformat_files
+from rosdistro.verify import reformat_files, verify_files_identical
```
Comment (dirk-thomas) [inline scripts/rosdistro_reformat]: Please use separate imports following PEP8.

Comment (mintar) [inline scripts/rosdistro_reformat]: Well, PEP8 says it's okay:

> Imports should usually be on separate lines, e.g.:
> 
> Yes:
>    import os
>    import sys
> 
> No:  import sys, os
> 
> It's okay to say this though:
> 
> from subprocess import Popen, PIPE

But I can of course change it, whatever you prefer!

Comment (dirk-thomas) [inline scripts/rosdistro_reformat]: Thanks for clarifying. Can you please still split the lines :wink: Thanks.

Comment (mintar) [inline scripts/rosdistro_reformat]: Done!

**Comment by dirk-thomas** [comment]
Thank you!

**Pull request merged**